convert2rhel/main.py

```diff
@@ -94,14 +88,22 @@ def main():
 
         ### End calls that should be put into pre_ponr_changes(), rollback(), or post_ponr_changes()
 
-        # begin conversion process
-        process_phase = ConversionPhase.PRE_PONR_CHANGES
-        # pre_ponr_conversion()
+        # FIXME: Write a nicely formatted report here.
+        # Write to the screen for this release
+        # Also write to a json formatted file for the next release.
```

Comment:
I think it is better to move this to a jira issue rather than a comment in the code if we already know what will be done for now.

Reply:
Done: https://issues.redhat.com/browse/RHELC-975